src/main/asciidoc/_chapters/getting_started.adoc

```diff
@@ -426,7 +386,7 @@ You can stop HBase the same way as in the <<quickstart,quickstart>> procedure, u
 
 
 [[quickstart_fully_distributed]]
-=== Advanced - Fully Distributed
+=== Fully Distributed for Production
 
 In reality, you need a fully-distributed configuration to fully test HBase and to use it in real-world scenarios.
 In a distributed configuration, the cluster contains multiple nodes, each of which runs one or more HBase daemon.
```

Comment:
We should make clear in this paragraph that this quickstart covers a necessary but not sufficient topic for a system that is "production ready."

Just to be clear that we are only explaining how to get a distributed HBase on top of a distributed filesystem and there exist other topics that need to be considered for a production deployment. Folks should still read through the section "The Important Configurations" and should ensure they have monitoring of metrics and log aggregation, for example.

Reply:
Maybe drop my addition of "for Production" then? I thought it odd that our default operating configuration was flagged as "advanced". I agree there's more to a production deploy than just these configs. However, I also don't think it's our job to be prescriptive of what a production deployment looks like, re: monitoring, metrics, log aggregation, &c. I think anything more than a paragraph or two of "recommended" or "strongly encouraged" supporting infrastructure is beyond the scope of our document.